### PR TITLE
Add set commands for ValueID

### PR DESCRIPTION
**Pull request by dhylands**
r? @julienw

### CODE REVIEW

File: src/value_classes/value_id.rs [hunk 1]
```diff
@@ -265,6 +265,81 @@ impl ValueID {
...
+
+    pub fn set_list_selection_string(&self, value: &str) -> bool {
+        if self.get_type() == ValueType::ValueType_List {
+            if let Ok(c_string) = CString::new(value) {
+                let manager_ptr = unsafe { extern_manager::get() };
+                return unsafe { extern_manager::set_value_list_selection_string(manager_ptr, &self.as_ozw_vid(), c_string.as_ptr()) };
+            }
+        }
+        false
+    }
```
Comment (julienw) [inline src/value_classes/value_id.rs:358]: I'd prefer to returns `Result<(), ()>` instead of bool, this makes it easier to use with `try!` etc.

Comment (julienw) [inline src/value_classes/value_id.rs:358]: see `res_to_result` in openzwave-sys/src/utils.rs, I use it in manager.rs and options.rs so far.

**Comment by dhylands** [comment]
Changed rust API functions to return Result<(), ()> rather than bool

**Comment by julienw** [comment]
r+

I couldn't make it work locally yet, but we can figure it out later

**Comment by dhylands** [comment]
It seems that it may have to wait for the next polling interval before the
command actually takes effect. I think that the interval is set to a minute.

On Thu, Mar 17, 2016 at 10:31 AM, Julien Wajsberg notifications@github.com
wrote:

> Merged #7 https://github.com/fxbox/openzwave-rust/pull/7.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/fxbox/openzwave-rust/pull/7#event-593773450

## 

Dave Hylands
Shuswap, BC, Canada
http://www.davehylands.com

**Pull request merged**